includes/common.php

```diff
@@ -1128,21 +1126,21 @@ function inet6_ntop($ip)
 }
 
 /**
- * Convert IP to use sysName
+ * If hostname is an ip, use return sysName
  * @param array device
- * @param string ip address
+ * @param string hostname
  * @return string
 **/
-function ip_to_sysname($device, $ip)
+function process_hostname($device, $hostname)
```

Comment:
You could make the second argument optional, that would make most of the calls cleaner.